org.vim.Vim.json

```diff
@@ -1,8 +1,9 @@
 {
   "app-id": "org.vim.Vim",
   "runtime": "org.freedesktop.Platform",
-  "runtime-version": "1.6",
+  "runtime-version": "18.08",
   "sdk": "org.freedesktop.Sdk",
+  "branch": "stable",
```

Comment:
This is not needed (Flathub passes `--default-branch=stable`) and makes it harder to distinguish a locally-compiled development version from the Flathub release. I deliberately removed it in e672374242b510ef3000db86460c0c078fb8a8b8.

Reply:
Ok, feel free to remove this or I could do it later.